intbot/core/bot/main.py

```diff
@@ -10,13 +10,71 @@
 
 bot = commands.Bot(command_prefix="!", intents=intents)
 
+# Inbox emoji used for adding messages to user's inbox
+INBOX_EMOJI = "📥"
+
 
 @bot.event
 async def on_ready():
     print(f"Bot is ready. Logged in as {bot.user}")
     poll_database.start()  # Start polling the database
 
 
+@bot.event
+async def on_raw_reaction_add(payload):
+    """Handle adding messages to inbox when users react with the inbox emoji"""
+    if payload.emoji.name == INBOX_EMOJI:
+        # Get the channel and message details
+        channel = bot.get_channel(payload.channel_id)
+        message = await channel.fetch_message(payload.message_id)
+
+        # Create a new inbox item using async
+        item = InboxItem(
+            message_id=str(message.id),
+            channel_id=str(payload.channel_id),
+            channel_name=f"#{channel.name}",
+            server_id=str(payload.guild_id),
+            user_id=str(payload.user_id),
+            author=str(message.author.name),
+            content=message.content,
+        )
+        await item.asave()
+
+
+@bot.event
+async def on_raw_reaction_remove(payload):
+    """Handle removing messages from inbox when users remove the inbox emoji"""
+    if payload.emoji.name == INBOX_EMOJI:
+        # Remove the inbox item
+        items = InboxItem.objects.filter(
+            message_id=str(payload.message_id),
+            user_id=str(payload.user_id),
+        )
+        await items.adelete()
+
+
+@bot.command()
+async def inbox(ctx):
+    """Display a user's inbox items"""
```

Comment:
Can you explain it a bit better what does this do here? Based on the description it sounds like the bot would return me random messages from my inbox. If people want to use this command, they will probably check this docstring, so let's maybe write here the motivation and what exactly it does.